style.css

```diff
@@ -0,0 +1,225 @@
+/*==========================================================
+    FONTS
+==========================================================*/
+@import url('https://fonts.googleapis.com/css?family=Roboto+Condensed:300,400,700&display=swap');
+/*==========================================================
+    VARIABLES
+==========================================================*/
+:root {
+    --news-red: #f72210;
+    --ethic-white: #fff;
+}
+/*==========================================================
+    GENERAL TAGS
+==========================================================*/
+body {
+    font-family: Arial,Helvetica,sans-serif;
+}
+
+.hid {
+    display: none;
+}
+
+body.container {
+    max-width: 1290px;
+    overflow-x: hidden;
+}
+
+h1, h2, h3, h4, h5 {
+    font: bold 19px/19px "Roboto Condensed",sans-serif;
+}
+
+p {
+    margin: 0;
+}
+
+ul, ol {
+    margin: 0;
+    list-style: none;
+}
+
+.fs-1-3 {
+    font-size: 1.3rem;
+}
+
+.txt-red {
+    color: var(--news-red)
+}
+
+.max-1290 {
+    max-width: 1290px;
+}
+
+section.max-1290 {
+    max-width: 1600px !important;
+}
+
+.bg-secondary {
+    background-color: #f4f5f6 !important;
```

Comment:
Always look for a way to use [specificity](https://developer.mozilla.org/en-US/docs/Web/CSS/Specificity) before even considering `!important`.

Reply:
Done.